storm-client/src/jvm/org/apache/storm/messaging/netty/Context.java

```diff
@@ -65,20 +65,8 @@ public synchronized IConnection bind(String storm_id, int port) {
      * establish a connection to a remote server
      */
     public synchronized IConnection connect(String storm_id, String host, int port, AtomicBoolean[] remoteBpStatus) {
-        IConnection connection = connections.get(key(host, port));
```

Comment:
Please rename connections to `serverConnections` as it is only for Servers now.  Also it would be nice to change it back to a list or a set instead of a map, because we will no longer be using it as a cache, so we don't need to pull out a specific value any longer.  We just want it so we can close them on termination.